src/chainlit/input_widget.py

```diff
@@ -220,3 +191,36 @@ def to_dict(self) -> Dict[str, Any]:
             "tooltip": self.tooltip,
             "description": self.description,
         }
+
+
+@dataclass
+class Tags(InputWidget):
+    """Useful to create an input for an array of strings."""
+
+    type: InputWidgetType = "tags"
+    initial: List[str] = Field(default_factory=lambda: list)
+    values: List[str] = Field(default_factory=lambda: list)
+
+    def __init__(
+        self,
+        key: str or int,
```

Comment:
Do we want to allow int for the other widget keys as well? When we get the settings back from the front-end is the key still an integer or is it converted to a string by formik?

Reply:
I originally looked at [streamlit](https://docs.streamlit.io/library/api-reference/widgets/st.text_input) to find inspiration about which properties should be supported and they allow the key to be both int or string, but apparently forgot to remove it properly since I found it confusing

Reply:
Ok, thanks for sharing your process! In that case it makes sense to move to string-only 👌